sijoittelu-service/src/test/java/fi/vm/sade/sijoittelu/HakijaryhmaTest.java

```diff
@@ -18,244 +16,252 @@
 
-        List<Hakukohde> hakukohteet = haku.getHakukohteet().parallelStream().map(DomainConverter::convertToHakukohde).collect(Collectors.toList());
-        final SijoitteluajoWrapper sijoitteluAjo = createSijoitteluajoWrapper(hakukohteet);
-        SijoittelunTila s = SijoitteluAlgorithmUtil.sijoittele(sijoitteluAjo);
-        sijoitteluAjo.setKaikkiKohteetSijoittelussa(LocalDateTime.now().plusDays(10));
+    List<Hakukohde> hakukohteet = haku.getHakukohteet().parallelStream().map(DomainConverter::convertToHakukohde).collect(Collectors.toList());
+    final SijoitteluajoWrapper sijoitteluAjo = createSijoitteluajoWrapper(hakukohteet);
+    SijoittelunTila s = SijoitteluAlgorithmUtil.sijoittele(sijoitteluAjo);
+    sijoitteluAjo.setKaikkiKohteetSijoittelussa(LocalDateTime.now().plusDays(10));
 
-        LOG.info("\r\n{}", tulostaSijoittelu(s));
-        assertoiAinoastaanValittu(hakukohteet.get(0).getValintatapajonot().get(0), "hakija1");
+    LOG.info("\r\n{}", tulostaSijoittelu(s));
+    assertoiAinoastaanValittu(hakukohteet.get(0).getValintatapajonot().get(0), "hakija1");
 
-    }
+  }
 
-    @Test
-    @UsingDataSet(locations = "ylitaytto_simple_case.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
-    public void testYlitayttoRekursio() {
 
-        HakuDTO haku = valintatietoService.haeValintatiedot("1.2.246.562.29.173465377510");
+  @Test
+  @Sql("ylitaytto_simple_case.sql")
+  public void testYlitayttoRekursio() {
 
-        List<Hakukohde> hakukohteet = haku.getHakukohteet().parallelStream().map(DomainConverter::convertToHakukohde).collect(Collectors.toList());
+    HakuDTO haku = valintatietoService.haeValintatiedot("1.2.246.562.29.173465377510");
 
-        final SijoitteluajoWrapper sijoitteluAjo = createSijoitteluajoWrapper(hakukohteet);
-        sijoitteluAjo.setKaikkiKohteetSijoittelussa(LocalDateTime.now().plusDays(10));
-        SijoittelunTila s = SijoitteluAlgorithmUtil.sijoittele(sijoitteluAjo);
+    List<Hakukohde> hakukohteet = haku.getHakukohteet().parallelStream().map(DomainConverter::convertToHakukohde).collect(Collectors.toList());
 
-        LOG.info("\r\n{}", tulostaSijoittelu(s));
+    final SijoitteluajoWrapper sijoitteluAjo = createSijoitteluajoWrapper(hakukohteet);
+    sijoitteluAjo.setKaikkiKohteetSijoittelussa(LocalDateTime.now().plusDays(10));
+    SijoittelunTila s = SijoitteluAlgorithmUtil.sijoittele(sijoitteluAjo);
 
-        assertoiAinoastaanValittu(hakukohteet.get(0).getValintatapajonot().get(0), "hakija3", "hakija4", "hakija5");
+    LOG.info("\r\n{}", tulostaSijoittelu(s));
 
-    }
+    assertoiAinoastaanValittu(hakukohteet.get(0).getValintatapajonot().get(0), "hakemus3", "hakemus4", "hakemus5");
+
+  }
 
-    @Disabled
-    @Test
-    @UsingDataSet(locations = "ylitaytto_vaihe.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
-    public void testYlitaytto() {
+  @Disabled
+  @Test
+  //@UsingDataSet(locations = "ylitaytto_vaihe.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
```

Comment:
Pitäiskö näitä disabloituja merkata lisäksi TODO:lla jos on tarkoitus fiksata joskus (TM)?

Reply:
Juu, tarkoitus oli fiksata niitä kun saisi tuon pääjutun eteenpäin. Lähinnä disabloitu sellaiset joiden testidatasetin konvertoimisessa on enemmän dataa